acs-core/src/main/java/com/arrow/acs/client/model/AccessKeyModel.java

```diff
@@ -9,46 +9,118 @@
  *     Arrow Electronics, Inc.
  *******************************************************************************/
 package com.arrow.acs.client.model;
+import java.util.ArrayList;
+import java.util.List;
 
-import java.io.Serializable;
-
-public class AccessKeyModel implements Serializable {
+public class AccessKeyModel extends ModelAbstract<AccessKeyModel> {
     private static final long serialVersionUID = 2009328107616383251L;
```

Comment:
In general, it is desired to re-generate serialVersionUID if change class fields